app/apollo/models/resource.schema.js

```diff
@@ -43,13 +43,20 @@ const ResourceSchema = new mongoose.Schema({
     default: false,
   },
   searchableData: {
-    type: Map,
+    type: Object,
     default: {},
   },
   searchableDataHash: {
     type: String,
     default: '',
   },
+  searchableDataHist: {
+    type: Object,
+    default: {},
+  },
+  histId: {
+    type: String,
+  },
```

Comment:
Why are these additional database fields needed now?  They weren't used previously (obviously, since they didn't exist) and I do not see any changes using them in this PR either.